src/locales/en.json

```diff
@@ -0,0 +1,6 @@
+{
+    "welcome": {
```

Comment:
Maybe change this for `waiting` because that's the screen that the code refers to (appstatus = waiting)